src/pageObject.js

```diff
@@ -1,16 +1,40 @@
 'use strict';
 
 class Header {
-
-};
+  get logo() {
+    return 'Conduit logo';
+  }
+}
 
 class PageObject {
+  constructor(url) {
+    this.url = url;
+  }
+  header = new Header();
```

Comment:
Header class instance should be created in the constructor.

Reply:
Fixed in 'Add fixes'